### ISSUE TITLE
Import all good content to ocamlverse

### ISSUE THREAD

**Issue by bluddy** [bluddy/ocamlverse#23]
What we're already beginning to see is that linking to outside content such as tutorials and such is not great. Since we don't have control over the content, we can't modify it, fix it or update it. An ecosystem evolves, and so should its documentation.

Therefore, I think we have to gradually move almost all content to OCamlverse itself. Linking should be seen as a temporary measure, either until we ask the author's permission, or until we recreate the content here. We could leave the link to the original content at the end of the article as a way to give credit, and we should definitely give credit to people when we import their content.

**Comment by pmetzger** [comment]
Content we bring in would need to have three qualities:

1. Appropriately licensed so we can use it (or the owners give us permission.)
2. Largely abandoned where it is (if something is well maintained where it is, we shouldn't be taking it in.)
3. Something we actually have interest in fixing up (because obviously there's no point in adding something we'll also leave abandoned.)

**Comment by bluddy** [comment]
Completely agree with 1 and 2.

For the third point, I think we can assume we'll want to fix up *all* content, if we consider it relevant, which is why we have it in the first place. That doesn't mean that our current crew is *capable* of doing that, and we should import everything right away, but the point of the site is that we're documenting and updating things. So I'd change that point to say

3. We should only import things we think we're realistically able to maintain.

**Comment by pmetzger** [comment]
> We should only import things we think we're realistically able to maintain.

Agreed on that wording. Maybe we should even put these rules up somewhere.
